src/main/resources/com/google/api/codegen/ruby/test.snip

```diff
@@ -0,0 +1,221 @@
+@extends "ruby/method_sample.snip"
+@extends "ruby/common.snip"
+@extends "ruby/initcode.snip"
+
+@snippet generate(apiTest)
+  {@header(apiTest.fileHeader)}
+
+  {@helpers()}
+
+  {@testClass(apiTest.testClass)}
+@end
+
+@private header(fileHeader)
+  {@license(fileHeader)}
+
+  {@importList(fileHeader.importSection.standardImports)}
+
+  {@importList(fileHeader.importSection.appImports)}
+@end
+
+@private helpers()
+  class CustomError < StandardError; end
+
+  class MockGrpcStub
+    def initialize(expected_symbol, mock_method)
+      @@expected_symbol = expected_symbol
+      @@mock_method = mock_method
+    end
+
+    def method(symbol)
+      return @@mock_method if symbol === @@expected_symbol
+      proc do
```

Comment:
When the client is initialized, the proc for each method of a service is wrapped by `Google::Gax#create_api_call`. Since we only want to override a specific method, we need to make sure that none of the other methods are called. For methods that aren't being tested. This raise proc is what is wrapped by `Google::Gax#create_api_call` and calling the method will raise this call.